recipe/gen_patch_json.py

```diff
@@ -1234,7 +1234,7 @@ def _gen_new_index(repodata, subdir):
             
         # Retroactively pin a max version of openlibm for julia 1.6.* and 1.7.*:
         # https://github.com/conda-forge/julia-feedstock/issues/169
-        if record_name == "sphinx" and (record["version"].startswith("1.6") or record["version"].startswith("1.7")):
+        if record_name == "julia" and (record["version"].startswith("1.6") or record["version"].startswith("1.7")):
```

Comment:
`str.startswith` can take a tuple of `str` and returns `True` if any of those is true.

```suggestion
        if record_name == "julia" and record["version"].startswith(("1.6", "1.7")):
```

Reply:
Also, we might want to add a timestamp check too, so future packages (hopefully corrected) are not affected.

Reply:
@jaimergp pls check timestamp addition, thanks